pygments/lexers/smithy.py

```diff
@@ -71,8 +71,8 @@ class SmithyLexer(RegexLexer):
             (r'\{', Text, "#push"),
             (r'\}', Text, "#pop"),
             (r'"{3}(\\\\|\n|\\")*"{3}', String.Doc),
+            (r'"""(?:.|\n)*?[^\\]"""', String.Double),
```

Comment:
Is an empty string not allowed? This regex always requires at least one character inside it.

Also, see `Contributing.md` about lexing strings: you probably want a dedicated state, so as to highlight string escapes.